okta_rules/okta_api_key_created.py

```diff
@@ -1,4 +1,7 @@
-from panther_base_helpers import deep_get, okta_alert_context  # pylint: disable=import-error
+from panther_base_helpers import (  # pylint: disable=import-error
```

Comment:
I know you didn't add it, but we ignore `import-error` everywhere now so I think we can remove this comment.